src/main/java/recipeditor/recipe/Ingredient.java

```diff
@@ -58,10 +44,14 @@ public static Ingredient parsedIngredients(String input) throws ParseException {
         String[] parsed = input.split(DIVIDER, MAX_DIVIDES);
         try {
             double amount = Double.parseDouble(parsed[1]);
+            if (parsed[0].replaceAll(" ", "").equals("")
```

Comment:
These seems to have magic literals in here. Maybe you can create a named string or char here instead.
Also, this line does seem a little complicated. Maybe you want to create functions to make this line more readable at one glance.

Reply:
This line is to convert an empty input or input consisting of `space` to an empty string, which makes checking arguments easier. I will shorten this!

Reply:
Your intention for this part of the code is good! However, it will be better to make it more readable! :>